Frontend/src/api/helper/context/permission_context.ts

```diff
@@ -0,0 +1,14 @@
+import { createContext } from 'react'
+import { Permissions } from '../../auth/get_permissions'
+
+interface PermissionsContext {
+  permissions?: Permissions
+  canAdminCompetition: (id: string) => boolean
+  canAttendCompetition: () => boolean
```

Comment:
Is this meant to be `canAttendCompetitions` (plural)? And below.
Why is it a function instead of just a boolean?

Reply:
Currently it could be a boolean, but I am adding conditions to the permissions route soon (e.g if the competitor is banned for competitions before a date) for which it needs to be a function

Reply:
done